docs/reference/supported-environments.md

```diff
@@ -4,6 +4,12 @@ title: "Supported environments"
 description: "Find out where to run Camunda 8 components for SaaS and Self-Managed, including Optimize for both Camunda 8 and Camunda 7."
 ---
 
+The supported environments page lists tested and compatible environments, deployment options, and component dependencies for successfully running Camunda 8.
```

Comment:
This sentence reads strange to me. What about?

```suggestion
The supported environments page lists environments, deployment options, and component dependencies, which are tested and supported for compatibility with Camunda 8.
```

Reply:
either way is fine for me